CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 - fix: run original function in runInAsyncContext/runPromiseInAsyncContext when tracing is not active.
+- Introduction of AWS SQS instrumentation for sending and reading messages
```

Comment:
Please add full stop at the end. Yes, I am very German and nitpicky, thanks for asking :-)